src/mixer/state.ts

```diff
@@ -529,6 +555,10 @@ export const load = (
       }
     });
     playerInstance.on("finish", () => {
+      // PFL on PFL Player
+      if (player === PLAYER_PFL_ID) {
+        dispatch(setChannelPFL(player, false));
+      }
```

Comment:
Eh?

Reply:
Mmm, I'll give this a better comment. Essentially the player still uses the same PFL logic to the headphones that regular channels do, but in this case, we have no PFL button, it's controlled by if the player is playing or not